packages/ui/src/components/shared/export/JsonFile.tsx

```diff
@@ -29,21 +38,28 @@ export default function JsonFile({ value, object }: JsonFileProps): JSX.Element
   };
 
   return (
-    <div className='text-center'>
-      <p className='mt-4 sm:px-20'>
-        {t<string>(
-          `Export this {{object}} to a JSON file and import it back to Coong Wallet on this or other devices later`,
-          { object: t<string>(object.toLowerCase()) },
-        )}
-      </p>
-      <Button onClick={downloadJsonFile} startIcon={<Download />}>
-        {t<string>('Download JSON File')}
-      </Button>
-      <p className='mt-4 italic sm:px-20 text-sm'>
-        {t<string>(`You will be prompted to enter your wallet password to complete importing the {{object}}`, {
-          object: t<string>(object.toLowerCase()),
-        })}
-      </p>
-    </div>
+    <>
+      <h3>{t<string>(title)}</h3>
+      {topInstruction || (
+        <p className='mt-4 sm:px-20 text-center'>
+          {t<string>(
+            `Export this {{object}} to a JSON file and import it back to Coong Wallet on this or other devices later`,
+            { object: t<string>(object.toLowerCase()) },
+          )}
+        </p>
```

Comment:
I don't think this is a good idea to set up default values for top/bottom instruction. One other way is to create an intermediate component that use those default values by default.
```tsx
export default function ExportJsonFile() {
  return (<JsonFile ... topInstruction={<p>Export this ....</p>} bottomInstruction={<p>Default instruction goes here!</p>} />)
}
```

Similarly, we should address this for the QR Code case as well.